### ISSUE TITLE
[bug]: File Upload Works Locally But Stuck in "Call unsuccessful..." in Vercel Development

### ISSUE THREAD

**Issue by alpacaPwaa** [pingdotgg/uploadthing#621]
### Provide environment information

```bash
System:
    OS: Windows 11 10.0.22621
    CPU: (4) x64 Intel(R) Core(TM) i3-8145U CPU @ 2.10GHz
    Memory: 370.51 MB / 7.86 GB
  Binaries:
    Node: 20.11.0 - C:\Program Files\nodejs\node.EXE
    npm: 9.6.2 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.15.3 - ~\AppData\Roaming\npm\pnpm.CMD
  Browsers:
    Chrome: 121.0.6167.185
    Edge: Chromium (121.0.2277.128)
    Internet Explorer: 11.0.22621.1
  npmPackages:
    @uploadthing/react: ^6.2.1 => 6.2.1
    typescript: ^5 => 5.3.3
    uploadthing: ^6.4.0 => 6.4.0
```

### Describe the bug

I'm stuck in "Call unsuccessful after 8 tries. Retrying in 64 seconds". Everything works fine in my localhost deployment but break in the vercel app deployment. The error only appears in larger files, specifically, files that are above 1MB, uploading smaller files works fine. I've already updated my upload thing to the latest version generated an updated API key, as well as disabled my vercel authentication, and redeployed my app multiple times. The file is being uploaded in my upload thing files and I can also see the file appear in my app dashboard.

![Screenshot (48)](https://github.com/pingdotgg/uploadthing/assets/142567568/70a6e753-44a8-45b8-966b-3adc81cf7600)
![Screenshot (51)](https://github.com/pingdotgg/uploadthing/assets/142567568/ea5ddd69-8bba-4021-a996-8f574644d9d3)

### Link to reproduction

Issue is with the deployment

### To reproduce

Issue is with the deployment

### 👨‍👧‍👦 Contributing

- [X] 🙋‍♂️ Yes, I'd be down to file a PR fixing this bug!

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by markflorkowski** [comment]
> The error only appears in larger files, specifically, files that are above 1MB

This sounds suspiciously close to the Vercel edge runtime [payload size limit](https://vercel.com/docs/functions/limitations). 

Without being able to see your code it is difficult to comment further, but are you potentially trying to download files in an edge function or something?

**Comment by alpacaPwaa** [comment]
I'm sorry but I'm unfamiliar with edge functions, but the entire application uses node js. The code is in my [repo](https://github.com/alpacaPwaa/askPDF), I'm just a beginner and still needed assistance and I really wanted to use uploadthing in my app so I'll really appreacite if y'll can help me fix this problem.

**Comment by markflorkowski** [comment]
Do you see any errors in your Vercel logs?

**Comment by alpacaPwaa** [comment]
No, I've just had redeployed the project and see no logs in my vercel build. 
![Screenshot (55)](https://github.com/pingdotgg/uploadthing/assets/142567568/05a39e3b-84d0-4c04-846a-02217abb88eb)

**Comment by alpacaPwaa** [comment]
I've tried uploading other files and it seems like larger files in general don't work, I've tested the app, and files that are 10 pages and above are uploaded fine.

**Comment by ThiagoMaurat** [comment]
Same here on self-hosted, vercel works...

**Comment by markflorkowski** [comment]
> Same here on self-hosted, vercel works...

@ThiagoMaurat This sounds like the opposite of the issue OP was facing. Is this happening when running a development or production build locally?
